### PR TITLE
Rebar3

### PR DESCRIPTION
**Pull request by puzza007**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,28 @@
-all: compile
+REBAR?=rebar3
 
-compile:
-	@ ./rebar compile
+.PHONY: \
```
Comment (AeroNotix) [inline Makefile:3]: update, dialyzer and coveralls aren't in .PHONY.

File: c_src/Makefile [hunk 2]
```diff
@@ -0,0 +1,67 @@
+CURDIR := $(shell pwd)
```
Comment (AeroNotix) [inline c_src/Makefile:1]: This Makefile pleases me

Comment (puzza007) [inline c_src/Makefile:1]: Generated by rebar3. Pretty nice!

Comment (AeroNotix) [inline c_src/Makefile:1]: Aint half bad!

**Pull request merged**